### DISCUSSION TITLE
JupyterHub Permissioning

### DISCUSSION THREAD

**Discussion by costrouc** [nebari-dev/nebari#239] [Ideas]
JupyterHub's current permissions model is:
  - is_admin?

We need a more rich permissions model that could enable:
  - a user to share a server with a set of users and groups
  - preventing access to certain api resources for a given api token e.g. should only be able to poll metrics
  - enable potential real time collaboration by allowing several users to access the same jupyter server. This would provide a much easier mechanism for rtc since it would not require webrtc etc.

**Comment by costrouc** [comment]
See https://github.com/jupyterhub/jupyterhub/tree/rbac

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
